src/report/error_key.rs

```diff
@@ -27,6 +27,7 @@ pub enum ErrorKey {
     ImageFormat,
     Unneeded,
     Scopes,
+    /// This seems like it solves the same problem that `Confidence` solves.
```

Comment:
Yeah the purpose is to hide the StrictScopes ones behind a command-line flag. This could be the same flag that activates weak-confidence reports.

Basically StrictScopes will warn if you use a scope name that has not been previously defined. But it means I have to catch all places where the game engine predefines scopes. It also gets more easily confused by complicated scripted effects than the normal mode does.